src/DynamoMigratorExtension.cs

```diff
@@ -24,7 +24,8 @@ public string Info
                 "This extension will automate the process of converting XML Dynamo files to JSON based 2.0 files.\n"+
                 "Select a source directory which contains Dynamo files and press Migrate.\n"+
                 "Optionally, you can also specify a target directory.\n"+
-                "Files will be saved to the target directory, or if none is given the source files will be overwritten in the source directory.";
+                "Files will be saved to the target directory, or if none is given the source files will be overwritten in the source directory.\n"+
+                "Note: For clockwork package, it is recommended to migrate the files in batch's of 50-100 at once. Close the dynamo window and open it again before running the next batch of files.";
```

Comment:
Actually for archi-lab which has around 180 packages, the tool works just fine and doesn't take too long. I tried it out on couple of other packages but faced the crash with only the clockwork package. One thing I noticed was that some of the files are more than 40kb in clockwork package and other packages had smaller files. So may be opening and closing large files is causing it. I agree that on some systems it might be happening with other packages as well. I will update the note to larger packages.